release-notes/azure/v11.2.1.md

```diff
@@ -12,11 +12,13 @@ To upgrade, please contact your Solution Engineer.
 
 Below, you can find more details on components that were changed with this release.
 
-### azure-operator v3.0.0
+### azure-operator v3.0.1
 
 - Upgraded the Azure SDK and Service API endpoints.
+- Fixed workers' over-provisioning during cluster creation.
+- Modified to retrieve component versions from releases.
 
-### Kubernetes v1.16.7
+### Kubernetes v1.16.8
 
 - Updated from v1.16.3 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#changelog-since-v1166)
```

Comment:
```suggestion
- Updated from v1.16.3 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#changelog-since-v1167)
```

Reply:
Thank you!